app/modules/ibridge/README.md

```diff
@@ -0,0 +1,12 @@
+iBridge Module
+==============
+
+Details information about the iBridge in the Mac.
+
+Database:
+* ibridge_boot_uuid - VARCHAR(255) - Boot UUID of iBridge
```

Comment:
Please remove the ibridge prefix for the table names - too much duplication

Reply:
Prefixes removed